src/types.ts

```diff
@@ -385,12 +389,33 @@ export interface OpenSeaAsset extends Asset {
   transferFeePaymentToken: OpenSeaFungibleToken | null
 }
 
+export interface LastSale {
+  eventType: string
+  auctionType: string
+  totalPrice: string
+  transaction: Transaction
+}
+
+export interface Transaction {
+  id: number
```

Comment:
what's `id` used for? except for Account pks (to help with multiple blockchains in the future), we shouldn't need the pks of any other model in frontend serializers

Reply:
What is "Account pks"?

Reply:
The ids for Account models (OpenSeaAccount in this file). We can’t always identify an account by its address alone, so we’ll need the id